userguide/intro.rst

```diff
@@ -96,6 +96,11 @@ fixes.
 
 These major features are new in this version:
 
+* Change if an alert is shown when using a specific portion of space on
```

Comment:
This is not so much a change as a new thing. So:
`` * Add options to show warning and critical alerts when a dataset reaches a certain percent of the allocated quota. New fields... ``